### PR TITLE
Update proxy SHA

### PR DESCRIPTION
**Pull request by bianpengyuan**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by bianpengyuan** [comment]
fyi @mandarjog

File: pkg/envoy/proxy.go [hunk 1]
```diff
@@ -116,7 +116,6 @@ func (e *envoy) args(fname string, epoch int, bootstrapConfig string) []string {
 		"--parent-shutdown-time-s", fmt.Sprint(int(convertDuration(e.Config.ParentShutdownDuration) / time.Second)),
 		"--service-cluster", e.Config.ServiceCluster,
 		"--service-node", e.Node,
-		"--max-obj-name-len", fmt.Sprint(e.Config.StatNameLength),
```
Comment (rshriram) [inline pkg/envoy/proxy.go:119]: we dont need this anymore?

Comment (rshriram) [reply pkg/envoy/proxy.go:119]: I added this long time ago to deal with cluster names longer than 63 characters

Comment (bianpengyuan) [reply pkg/envoy/proxy.go:119]: Envoy removed it: https://github.com/istio/istio/issues/25595.

**Pull request merged**